lib/WP_Auth0_Routes.php

```diff
@@ -22,11 +37,6 @@ public function setup_rewrites( $force_ws = false ) {
 
 		add_rewrite_rule( '^auth0', 'index.php?auth0=1', 'top' );
 		add_rewrite_rule( '^\.well-known/oauth2-client-configuration', 'index.php?a0_action=oauth2-config', 'top' );
-
-		// if ( $force_ws || $this->a0_options->get( 'migration_ws' ) ) {
```

Comment:
This is no longer relevant?

Reply:
@cocojoe - It's been commented out for a while, no need to keep in in there.